fastlane/Fastfile

```diff
@@ -51,7 +51,14 @@ lane :automatic_bump do |options|
     github_rate_limit: options[:github_rate_limit]
   )
   options[:next_version] = next_version
-  UI.user_error!('Skipping automatic bump since the next version is a major release') if type_of_bump == :major
+  if type_of_bump == :skip
+    UI.error('Skipping automatic bump since the next version doesn\'t include public facing changes')
```

Comment:
I think it's better to use `UI.message` for these
```suggestion
    UI.message('Skipping automatic bump since the next version doesn\'t include public facing changes')
```

Reply:
Sure, I will update. I don't have a preference. Thanks for the suggestion!